corehq/apps/reports_core/templates/reports_core/base_template_new.html

```diff
@@ -16,11 +16,15 @@
     <script src="{% static 'reports/js/config.dataTables.bootstrap.js' %}"></script>
     <script src="{% static 'reports_core/js/choice_list_utils.js' %}"></script>
     <script src="{% static 'reports_core/js/charts.js' %}"></script>
-    <script src="{% static 'leaflet/dist/leaflet.js' %}"></script>
-    <script src="{% static 'reports/js/maps_utils.js' %}"></script><!-- depends on leaflet -->
-    <script src="{% static 'reports_core/js/maps.js' %}"></script>
     <script src="{% static 'userreports/js/report_analytix.js' %}"></script>
   {% endcompress %}
+
+  {# don't compress this block related to maps. it causes strange issues with the default image path #}
```

Comment:
This might be a good place to link to jira, so we could potentially try compressing these again when/if we upgrade leaflet - at least then we'd know what to test for.